app/test_main.py

```
@@ -1,3 +1,20 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ]
+)
+def test_get_human_age(cat_age: int, dog_age: int, expected) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

The suite lacks tests for incorrect input types (e.g., strings, floats, None) asserting that the correct exception is raised. This violates: "Incorrect input types should raise the correct exception (tests should assert exception type). [CHECKLIST ITEM #2]" Add parameterized tests using pytest.raises(...) to assert the expected exception type when `get_human_age` is called with invalid types.